Readme.md

```diff
@@ -95,6 +95,16 @@ Optionally, the following environment variables can be set:
 
 Once configured, pipelines can be migrated to GitHub Actions by opening an issue with the relevant issue template and following the instructions.
 
+### Custom Transformers
+
+Provided in this repository is an example [transformers.rb](./transformers.rb) which can be used in combination with `dry-run` and `migrate` actions to provide extensibility for customizing transformations from imported pipeline configurations.
+
+Example usage with Jenkins:
+
+```sh
+/dry-run --source-url :jenkins-job-url --custom-transformers transformers.rb
```

Comment:
Do you anticipate there being a command that you _wouldn't_ use custom transformers for? 

If not, it may be a simpler UX to define a path convention for where customer transformers live. For example, if any files exist in the `transformers/` directory of the repo, we could automatically append `--custom-transformers transformers/**/*.rb` to Actions Importer commands. That way, downstream users of the issue ops workflow don't really need to know about custom transformers 💭

Reply:
oh, I like that path _MUCH_ better, let me retrofit for that

Reply:
I guess the potential here perhaps is if you'd want to specify different transformers for different sources -- say you wanted to overwrite all of the build agents for one workflow/job but not another, then it might help to just pass the argument with a targeted transformer rb file instead.

I do like the idea of making it a bit more auto-magic and if a file is there it will get read, the only hiccup i see to that is if a larger or more complex org might want to have multiple transformers available for different things.  It'd be easy enough to make the logic map by source types so maybe `transformers/jenkins/**/*.rb` or `transformers/circle-ci/**/*.rb` would pick up source appropriate files, not sure if that starts making it _more_ confusing than a command line switch at that point.

Reply:
There's not a ton of overlap with the naming convention of items across the different platforms so generally it's fine to use custom transformers meant for one platform for another. (For example, Azure DevOps names tasks quite different to CircleCI, etc). 

One of the principles of this template is that it addresses ~95% of use cases, while allowing more advanced users to customize it if needed.

Reply:
Ah, yep I wasn't thinking of that at all, anyone could modify as needed for their use case and if they are using custom transformers then most likely modifying code in this repo to suit needs is a trivial matter, good call!  Will push up auto-parsing-transformers logic tomorrow

Reply:
I went for the "why not both?" approach -- added so that it will auto read any rb files under `./transformers/` but will allow you to override and pass a single file with comment switch if you don't want to read them all so folks can have a bunch of files but choose one if needed.